lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -266,6 +273,16 @@
             $( "#webcam_capture_button", this.el ).removeClass('is-hidden');
```

Comment:
@aamir-khan for all of these selectors you are including this.el too, can you please explain why? It appears that you just need the first selector, or to do something like 
```
this.$el.find(#webcam_capture_button)...
```
Also, it makes sense to assign variables to these elements from a performance point of view (in this example you are creating 3 of the same jQuery elements when you could create one and re-use it). e.g.
```
var $resetBtn = this.$el.find('#webcam_reset_button');
$resetBtn.on( 'click', ......);
```